scripts/variantstore/wdl/ngs_filter_extract.wdl

```diff
@@ -402,6 +412,7 @@ task IndelsVariantRecalibrator {
       -an ~{sep=' -an ' recalibration_annotation_values} \
       ~{true='--use-allele-specific-annotations' false='' use_allele_specific_annotations} \
       -mode INDEL \
+       ~{model_report_arg} \
```

Comment:
I think this can just be

```
~{"--input model " + model_report}
```
and you can remove the `model_report_arg` and `MODEL_REPORT` declarations

Reply:
very nice, thanks!